src/imp/linux_raw/conf/auxv.rs

```diff
@@ -18,26 +18,26 @@ use linux_raw_sys::general::{
     AT_SYSINFO_EHDR,
 };
 
-#[cfg(feature = "process")]
+#[cfg(feature = "conf")]
 #[inline]
 pub(crate) fn page_size() -> usize {
     auxv().page_size
 }
 
-#[cfg(feature = "process")]
+#[cfg(feature = "conf")]
 #[inline]
 pub(crate) fn clock_ticks_per_second() -> u64 {
     auxv().clock_ticks_per_second as u64
 }
 
-#[cfg(feature = "process")]
+#[cfg(feature = "conf")]
 #[inline]
 pub(crate) fn linux_hwcap() -> (usize, usize) {
     let auxv = auxv();
     (auxv.hwcap, auxv.hwcap2)
 }
 
-#[cfg(feature = "process")]
+#[cfg(feature = "conf")]
```

Comment:
What is the difference between `linux_execfn` and `exe_phdrs` that requires different feature?
Is there a situation where the `runtime` feature's function should not be used but the `conf` feature's function should?

Reply:
"runtime" is for APIs that require coordination with the process runtime (libc or mustang) and is a hidden API. Most rustix users do not use it.

For some things, the distinction is clear. Here, I guess I arbitrarily decided that `linux_execfn` is suitable to be a public API, and `exe_phdrs` is not. If anyone has a use case for `exe_phdrs`, we could change it.

Reply:
Okay cool, that makes much more sense. The appearance of the runtime feature in the `auxv` boggled me.

Reply:
Ah, another reason is that if someone does need something from `exe_phdrs`, it might be nice to expose a dedicated and possibly safe API for whatever is needed, rather than just handing out the raw ELF phdrs.